### PR TITLE
Feature/shopping cart and game detail images

### PR DESCRIPTION
**Pull request by Tiewhan**
<h1>Overview</h2>
The final image logic is done for the Game Details and Shopping Cart

<h2>Notes</h2>
This PR is dependent on PR #26 . Therefore a lot of changes will be coped over. The changes by this PR will be marked by a comment.

### CODE REVIEW

File: Example/Tests/GameViewTests/GameDetailsTests/MockGameDetailsLoadedType.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ class MockGameDetailsLoadedType: NSObject, GameDetailsLoadedType {
   var gameDetailsFoundCalled: Bool = false
```
Comment (Tiewhan) [inline Example/Tests/GameViewTests/GameDetailsTests/MockGameDetailsLoadedType.swift:14]: This PR Changed this file

File: Source/CoreClasses/DataTransferObjects/GameDataTransferObject.swift [hunk 2]
```diff
@@ -13,8 +13,9 @@ public struct GameDataTransferObject {
   public var name: String
```
Comment (Tiewhan) [inline Source/CoreClasses/DataTransferObjects/GameDataTransferObject.swift:13]: This PR Changed this file

File: Source/GameDetails/ObjCGameDetails/GameDetailsLoadedType.h [hunk 3]
```diff
@@ -15,7 +15,8 @@ NS_ASSUME_NONNULL_BEGIN
                         : (NSString *) price
```
Comment (Tiewhan) [inline Source/GameDetails/ObjCGameDetails/GameDetailsLoadedType.h:15]: This PR Changed this file

File: Source/GameDetails/ObjCGameDetails/GameDetailsViewModel.m [hunk 4]
```diff
@@ -40,6 +40,7 @@ - (void)getGameData {
   NSString* shortDescription = self.selectedGame.shortDescription;
```
Comment (Tiewhan) [inline Source/GameDetails/ObjCGameDetails/GameDetailsViewModel.m:40]: This PR Changed this file

**Review by bgouws — APPROVED** [review]
Looks good to me.

**Review by AkuaQ — APPROVED** [review]

**Pull request merged**